### PR TITLE
feat(*): add GitHub star

### PR DESCRIPTION
**Pull request by Leopoldthecoder**
- adds a GitHub star in the navbar, [KAG-2089](https://konghq.atlassian.net/browse/KAG-2089)
- adds issue template
- adds PR template

The templates are based on those of the `kong/kong` repo.

[KAG-2089]: https://konghq.atlassian.net/browse/KAG-2089?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Feature Request
+    url: https://github.com/Kong/kong/discussions/categories/ideas
```
Comment (Leopoldthecoder) [inline .github/ISSUE_TEMPLATE/config.yml:4]: Question: do we need to enable `Discussions` for this repo? My concern is that if we do, users will sometimes discuss Gateway-related(not GUI-related) topics here.

Comment (raoxiaoyan) [reply .github/ISSUE_TEMPLATE/config.yml:4]: I don't think there's any problem; If it is really to discuss kong related issues, can we push it to the Kong side?
What do you think?

Comment (sumimakito) [reply .github/ISSUE_TEMPLATE/config.yml:4]: That's true. However, we may utilize [GitHub's transferring discussions feature](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#transferring-a-discussion) to move off-topic discussions to the right place.

Comment (Leopoldthecoder) [reply .github/ISSUE_TEMPLATE/config.yml:4]: Makes sense. Didn't know discussions can be transferred.

**Review by raoxiaoyan — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (Leopoldthecoder) [reply .github/ISSUE_TEMPLATE/config.yml:4]: I enabled `Discussions` for this repo and updated code accordingly.

Comment (sumimakito) [reply .github/ISSUE_TEMPLATE/config.yml]: Sounds good.

**Pull request merged**